a11y-meta-display-guide/2.0/draft/localizations/en-US/display_guide_vocabulary_w3c_en-US.json

```diff
@@ -6,269 +6,316 @@
-            "conformance-wcag-2-0-compact": " WCAG 2.0 ",
-            "conformance-wcag-2-2-descriptive": " Web Accessibility Content Guidelines (WCAG) 2.2 ",
-            "conformance-wcag-2-1-descriptive": " Web Accessibility Content Guidelines (WCAG) 2.1 ",
-            "conformance-wcag-2-0-descriptive": " Web Accessibility Content Guidelines (WCAG) 2.0 ",
-            "conformance-level-aaa": " Level AAA",
-            "conformance-level-aa": " Level AA",
-            "conformance-level-a": " Level A",
-            "conformance-certification-info-compact": "The publication was certified",
-            "conformance-certification-info-descriptive": "The publication was certified",
-            "conformance-certification-date-prefix": " on ",
-            "conformance-certifier-prefix-compact": " by ",
-            "conformance-certifier-prefix-descriptive": " by ",
-            "conformance-certifier-credentials-prefix-compact": " The certifier's credential is ",
-            "conformance-certifier-credentials-prefix-descriptive": " with a credential of ",
-            "conformance-certifier-report": "For more information refer to the certifier's report."
-        }
-    },
-    "pre-recorded-audio": {
-        "pre-recorded-audio-title": "Pre-recorded audio",
-        "pre-recorded-audio-only": {
-            "descriptive": "Audiobook with no text alternative",
-            "compact": "Audio only"
-        },
-        "pre-recorded-audio-complementary": {
-            "descriptive": "Contents available as complementary audio and text",
-            "compact": "Complementary audio and text"
-        },
-        "pre-recorded-audio-synchronized": {
-            "descriptive": "All the content is available as pre-recorded audio synchronized with text",
-            "compact": "Synchronized audio and text"
-        },
-        "pre-recorded-audio-no-metadata": {
-            "descriptive": "No information about pre-recorded audio",
-            "compact": "No information about pre-recorded audio"
+        "conformance-aaa": {
+            "compact": "This publication exceeds accepted accessibility standards",
+            "descriptive": "The publication contains a conformance statement that it meets the EPUB Accessibility and WCAG 2 Level AAA standard"
+        },
+        "conformance-certification-info": {
+            "compact": "The publication was certified on",
+            "descriptive": "The publication was certified on"
+        },
+        "conformance-certifier": {
+            "compact": "The publication was certified by",
+            "descriptive": "The publication was certified by"
+        },
+        "conformance-certifier-credentials": {
+            "compact": "The certifier's credential is",
+            "descriptive": "The certifier's credential is"
+        },
+        "conformance-certifier-report": {
+            "compact": "For more information refer to the certifier's report",
+            "descriptive": null
```

Comment:
Probably, instead of Null, we should reproduce the Compact.

Reply:
You can iterate over the json and automatically populate the null values with the compact ones at run time.

Or code the output so that you never attempt to use the descriptive statement in those cases that are null.

Or write some bloated checking to test if you need the descriptive statement and take the compact when it's null.

I can live with null values, in other words, although my preference would be to not have them. If it means we have to duplicate every string in the guidelines so they can be harvested, that might sway me the other way. But could the script that builds the json automatically insert the compact statement instead of null?

Reply:
@mattgarrish there is no need to have all the strings in the guidelines, I can fix the script that generates the JSON to use the same value for compact and descriptive, in case there is no descriptive

Reply:
I would double up the strings, then, rather than use null.

We're already repeating the strings in the json for the no information available outputs.